numpy/polynomial/tests/test_hermite.py

```diff
@@ -99,6 +101,15 @@ def test_hermdiv(self):
                 res = herm.hermadd(herm.hermmul(quo, ci), rem)
                 assert_equal(trim(res), trim(tgt), err_msg=msg)
 
+    def test_hermpow(self):
+        for i in range(5):
+            for j in range(5):
+                msg = "At i=%d, j=%d" % (i, j)
+                c = np.arange(i + 1)
```

Comment:
Does this still pass if you use `arange(i)`?

Reply:
No, I'll paste the message if you like